src/NuGet.Clients/NuGet.PackageManagement.UI/Options/CheckedListBoxAccessibleObject.cs

```diff
@@ -38,12 +39,22 @@ public override AccessibleObject GetChild(int index)
 
                 if (packageSource.IsHttp && !packageSource.IsHttps && !packageSource.AllowInsecureConnections)
                 {
-                    var sourceMessage = string.Concat(
-                        Resources.Warning_HTTPSource,
+                    var sourceMessage = string.Format(
+                        CultureInfo.CurrentCulture,
+                        Resources.Error_HttpSource_Single,
                         packageSource.Source);
                     return new CheckedListBoxItemAccessibleObject(this, packageSource.Name, index, sourceMessage);
                 }
 
+                if (packageSource.AllowInsecureConnections)
+                {
+                    var insecureConnectionMessage = string.Format(
+                        CultureInfo.CurrentCulture,
+                        Resources.Warning_HTTPSource,
+                        packageSource.Source);
+                    return new CheckedListBoxItemAccessibleObject(this, packageSource.Name, index, insecureConnectionMessage);
```

Comment:
nit: could just set the insecure message variable in this conditional, then only need one return below